nsxt/resource_nsxt_vpc_test.go

```diff
@@ -178,7 +178,7 @@ func testAccNsxtVpcExists(displayName string, resourceName string) resource.Test
 			return fmt.Errorf("Policy Vpc resource ID not set in resources")
 		}
 
-		exists, err := resourceNsxtVpcExists(testAccGetProjectContext(), resourceID, connector)
+		exists, err := resourceNsxtVpcExists(testAccGetMultitenancyContext(), resourceID, connector)
```

Comment:
Is this change (and the other similar ones) unrelated to this commit?

Reply:
I've renamed this helper as part of PR, for improved code readability.